examples/v1alpha2/simple-gateway/gateway.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Gateway
+apiVersion: gateway.networking.k8s.io/v1alpha2
+metadata:
+  name: prod-web
+spec:
+  gatewayClassName: acme-lb
+  listeners:  
+  - protocol: HTTP
+    port: 80
+    routes:
+      kind: HTTPRoute
+      selector:
+        matchLabels:
+          gateway: prod-web-gw
```

Comment:
All of this can be removed. It's actually no longer valid in v1alpha2.

Reply:
Isn't https://github.com/kubernetes-sigs/gateway-api/blob/master/apis/v1alpha2/gateway_types.go#L235
the definition ?

Reply:
Yep, that's the correct reference, but the `ListenerRoutes` type has changed significantly in v1alpha2: https://github.com/kubernetes-sigs/gateway-api/blob/master/apis/v1alpha2/gateway_types.go#L343.

Reply:
9e2cc369730aa65d34308c6c7f22bbf7ae8579ab